stdlib/_tkinter.pyi

```diff
@@ -86,6 +90,9 @@ class TkappType:
     def unsetvar(self, *args, **kwargs): ...
     def wantobjects(self, *args, **kwargs): ...
     def willdispatch(self): ...
+    if sys.version_info >= (3, 12):
+        def gettrace(self, /) -> _TkinterTraceFunc | None: ...
+        def settrace(self, func: _TkinterTraceFunc | None, /) -> None: ...
```

Comment:
These were added in https://github.com/python/cpython/commit/1ff626ebda465931ff3e4922e8e87d586eb6244c